integrations/whatsapp/src/conversation.ts

```diff
@@ -108,6 +125,30 @@ export const createConversation: botpress.IntegrationProps['createConversation']
       }`
     )
 
+  return conversation
+}
+
+/**
+ * This handler is for allowing to initiate a Whatsapp conversation using the Botpress Client.
```

Comment:
**question:** I'm not sure I understand the comment. Actions are also callable from the botpress client

Reply:
Maybe I'm not understanding correctly but it's for allowing to create conversations from bot-as-code, which is the current supported behavior of the Whatsapp integration, see:
https://docs.botpress.cloud/docs/cloud/channels/whatsapp/#creating-conversations